pkg/hub/manager.go

```diff
@@ -18,21 +18,28 @@ import (
 	"github.com/open-cluster-management/registration/pkg/hub/rbacfinalizerdeletion"
 
 	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/rest"
 )
 
 // RunControllerManager starts the controllers on hub to manage spoke cluster registration.
 func RunControllerManager(ctx context.Context, controllerContext *controllercmd.ControllerContext) error {
-	kubeClient, err := kubernetes.NewForConfig(controllerContext.KubeConfig)
+	// Increase the QPS and Burst to enhance the ability of kube client to hanle requstes requests in concurrent
+	// TODO: these may be tuned by the flag of library-go when new a controller command in the future
```

Comment:
change some wording here. Something like:

Use flag to change qps/burst

Reply:
changed